src/util.js

```diff
@@ -268,3 +268,9 @@ export function compare_versions(a, b) {
 
     return a_ints.length - b_ints.length;
 }
+
+export function isRootUser() {
+    return window.cockpit.user().then((user) => {
+        return user.name === 'root';
+    });
```

Comment:
This function isn't needed, but just FTR: This should be written as

    return window.cockpit.user().then(user => user.name === 'root');